### PR TITLE
Add desktop version of the Portfolio

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
- Fix linters errors
- Add README file
- Add html and css
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,119 +1,253 @@
...
+            <li><button type="button" class="html-btn btn">HTML</button></li>
+            <li><button type="button" class="css-btn btn">CSS</button></li>
+            <li><button type="button" class="js-btn btn">JavaScript</button>
+            </li>
+        </ul>
+        <button class="tonic-button-submit submit-btn" type="submit">See Projects</button>
+    </div>
+    </section>
+
+    <section class="work2">
```
Comment (Shaher-11) [inline index.html:75]: Increase the width of each project card

File: index.html [hunk 2]
```diff
@@ -1,119 +1,253 @@
...
+
+    </section>
+
+    <section class="work3">
+        
+        <div class="work33">
+            <img class="work1" src="./images/portfolio2.svg" alt="">
+        </div>
+        <h3 class="facebook">Facebook360</h3>
+        <div>
```
Comment (Shaher-11) [inline index.html:109]: You need to style the section and avoid the text overlapping 
### Original project
right
#### Your project
wrong

File: index.html [hunk 3]
```diff
@@ -1,119 +1,253 @@
...
+             </li>
+         </ul>
+         <button class="stories-button-submit submit-btn1" type="submit">See Projects</button>
+     </div>
+
+    </section>
+
+  <!-- aboutpage -->
+   <section class="about">
+       <div class="about-me">
```
Comment (Shaher-11) [inline index.html:161]: Nothing is showing in the about section
wrong

File: index.html [hunk 4]
```diff
@@ -1,119 +1,253 @@
...
+                    <rect width="117.53" height="21.0255" transform="matrix(0.801365 0.598176 0.598176 -0.801365 0.857422 17.606)" fill="#7F8CFF"/>
+                    </svg>
+            </div>
+    
+            
+      
+      <h6>Contact me</h6>
+      <p class="content">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <div class="form">
+        <form action="https://formspree.io/f/" method="post">
```
Comment (Shaher-11) [inline index.html:223]: You need to reduce the extra space around the section and center it as shown below
### Original design 
right

#### Your Project
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Isabellanamutenda

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html [hunk 5]
```diff
@@ -1,119 +1,253 @@
...
+                    <li><a href="#"><img  src="./images/Linkedinicon.svg" alt=""></a></li>
+                    <li><a href="#"><img  src="./images/medium.svg" alt=""></a></li>
+                    <li><a href="#"><img  src="./images/github.svg" alt=""></a></li>
+                    <li><a href="#"><img  src="./images/angellist.svg" alt=""></a></li>
+                 </ul>
+            </div>
+      
+       </div>
+     
+   </section>
```
[the remark this answers is not in the document]
Comment (Isabellanamutenda) [reply index.html:45]: Resolved

File: index.html (hunk 1 above)
Comment (Isabellanamutenda) [reply index.html]: Resolved

File: index.html (hunk 2 above)
Comment (Isabellanamutenda) [reply index.html]: Resolved

File: index.html (hunk 3 above)
Comment (Isabellanamutenda) [reply index.html]: Resolved

File: index.html (hunk 4 above)
Comment (Isabellanamutenda) [reply index.html]: Resolved

File: index.html [hunk 6]
```diff
@@ -2,118 +2,255 @@
...
+                                alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/belguith-taheny-47b93a162/"><img class="image2"
+                                src="./images/linkedin.svg" alt="linkedin"></a></li>
+                    <li><a href="https://medium.com/@tahnyybelguith"><img class="image3" src="./images/medium.svg"
+                                alt="medium"></a></li>
+                    <li><a href="https://github.com/TahenyBELGUITH"><img class="image5" src="./images/github.svg"
+                                alt="github"></a></li>
+                    <li><a href="#"><img class="image6" src="./images/angellist.svg" alt="angellist"></a></li>
+                </ul>
+            </div>
```
Comment (Shaher-11) [inline index.html:61]: The section is still not styled as the original design, 
see the difference below

### Original Design
right
### Your Project
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Isabellanamutenda,

### Great work on making the changes requested by the previous reviewer:clap::clap::clap:
You've done a good job implementing some of the requested changes but there are still some which aren't done yet,  so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [ ] Check the comment below:point_down:

## Optional Suggestions💡

_The following point is highly recommended to improve your code quality, Please consider implementing it.

- Increase the width of the contact form.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Isabellanamutenda,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html (hunk 6 above)
Comment (Isabellanamutenda) [reply index.html]: Resolved

**Pull request merged**